apps/astarte_housekeeping/lib/astarte_housekeeping/rpc/handler.ex

```diff
@@ -152,6 +153,47 @@ defmodule Astarte.Housekeeping.RPC.Handler do
     end
   end
 
+  defp call_rpc({:update_realm, %UpdateRealm{} = call}) do
+    %UpdateRealm{realm: realm_name} = call
```

Comment:
I don't think it's worth it extracting the realm name this way as

1. we're matching a single value. `call.realm` works just as well
2. we're already matching on `%UpdateRealm{}` in the function head, so this doesn't give us new information
3. `realm_name` is only used in the `Engine.update_realm` call. it's probably safe to just call `call.realm` there, or even consider refactoring `update_realm` to extract the realm name itself, depending on the expected future usage of that function.